src/Android/Renderers/CustomTabbedRenderer.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Android.Content;
+using Android.Views;
+using Bit.Droid.Renderers;
+using Google.Android.Material.BottomNavigation;
+using Xamarin.Forms;
+using Xamarin.Forms.Platform.Android;
+using Xamarin.Forms.Platform.Android.AppCompat;
+
+[assembly: ExportRenderer(typeof(TabbedPage), typeof(CustomTabbedRenderer))]
+
+namespace Bit.Droid.Renderers
+{
+    public class CustomTabbedRenderer : TabbedPageRenderer, BottomNavigationView.IOnNavigationItemReselectedListener
+    {
+        private TabbedPage _page;
+
+        public CustomTabbedRenderer(Context context) : base(context) { }
+
+        protected override void OnElementChanged(ElementChangedEventArgs<TabbedPage> e)
+        {
+            base.OnElementChanged(e);
+            if (e.NewElement != null)
+            {
+                _page = e.NewElement;
+                GetBottomNavigationView()?.SetOnNavigationItemReselectedListener(this);
+            }
+            else
+            {
+                _page = e.OldElement;
+            }
+        }
+
+        private BottomNavigationView GetBottomNavigationView()
+        {
+            for (var i = 0; i <= ViewGroup.ChildCount - 1; i++)
+            {
+                var childView = ViewGroup.GetChildAt(i);
+                if (childView is ViewGroup viewGroup)
+                {
+                    for (var j = 0; j <= viewGroup.ChildCount - 1; j++)
+                    {
+                        var childRelativeLayoutView = viewGroup.GetChildAt(j);
+                        if (childRelativeLayoutView is BottomNavigationView bottomNavigationView)
+                        {
+                            return bottomNavigationView;
+                        }
+                    }
+                }
+            }
+            return null;
+        }
+
+        async void BottomNavigationView.IOnNavigationItemReselectedListener.OnNavigationItemReselected(IMenuItem item)
```

Comment:
Any reason this interface is explicitly implemented here vs. generally implemented?

i.e.
```C#
async void BottomNavigationView.IOnNavigationItemReselectedListener.OnNavigationItemReselected(IMenuItem item)
// vs.
async void OnNavigationItemReselected(IMenuItem item)
```

Reply:
it's actually generally implemented from `Google.Android.Material.BottomNavigation` - the view itself (`BottomNavigationView`) isn't part of the namespace.